src/os/bluestore/BitAllocator.cc

```diff
@@ -1056,7 +1056,8 @@ void BitMapAreaLeaf::init(int64_t total_blocks, int64_t area_idx,
 
   init_common(total_blocks, area_idx, def);
   debug_assert(m_level == 1);
-  num_child = total_blocks / pow(BitMapArea::get_span_size(), m_level);
```

Comment:
@chhabaramesh For a leaf node, we assert **m_level**  to be 1, otherwise something is serious wrong.

So
```
num_child = total_blocks / pow(BitMapArea::get_span_size(), m_level);

```
can be simplified in the following way:

```
int span_size = BitMapArea::get_span_size();
num_child = total_blocks / span_size;